pandas/core/frame.py

```diff
@@ -2116,7 +2116,7 @@ def from_records(
         Convert structured or record ndarray to DataFrame.
 
         Creates a DataFrame object from a structured ndarray, sequence of
```

Comment:
```suggestion
        Creates a DataFrame object from a structured ndarray, or sequence of
```

(now the "sequence of tuples or dicts" is the last entry in the enumeration, I think we should add an "or" here for clarity?